### PR TITLE
Make stresslog work for libraries other than CoreClr (I need it for DBI)

### PR DESCRIPTION
**Pull request by Djuffin**
Introducing of PAL_GetPalModuleBase removes dependency on CoreCLR name for
stress log module address.

### CODE REVIEW

File: src/pal/inc/pal.h [hunk 1]
```diff
@@ -3340,6 +3340,14 @@ PALAPI
 LPCVOID
 PAL_GetCoreClrModuleBase();
```
Comment (jkotas) [inline src/pal/inc/pal.h]: We should delete this method. The one remaining use can be replaced by PAL_GetPalModuleBase.

File: src/pal/src/loader/module.cpp [hunk 2]
```diff
@@ -1950,3 +1950,47 @@ PAL_GetCoreClrModuleBase()
...
+LPCVOID
+PAL_GetPalModuleBase()
+{
+    LPCVOID retval = NULL;
+
+    PERF_ENTRY(PAL_GetPalModuleBase);
+    ENTRY("PAL_GetPalModuleBase\n");
+
+    void* current_func = PAL_GetPC();
+    if (current_func != NULL)
```
Comment (jkotas) [inline src/pal/src/loader/module.cpp]: You can just use address of PAL_GetPalModuleBase instead of all this

File: src/pal/inc/pal.h (hunk 1 above)
Comment (Djuffin) [inline src/pal/inc/pal.h]: Good point. Fixed.

**Comment by sergiy-k** [comment]
LGTM

File: src/pal/inc/pal.h (hunk 1 above)
Comment (jkotas) [inline src/pal/inc/pal.h]: I still see PAL_GetCoreClrModuleBase in the latest diff. Was it really deleted?

File: src/pal/src/loader/module.cpp (hunk 2 above)
Comment (Djuffin) [inline src/pal/src/loader/module.cpp]: Good point. Fixed.

File: src/pal/inc/pal.h (hunk 1 above)
Comment (Djuffin) [inline src/pal/inc/pal.h]: Not it is. Sorry, I replied to the wrong comment before that.

**Comment by jkotas** [comment]
LGTM

**Pull request merged**